pandas/tests/test_index.py

```diff
@@ -41,6 +41,18 @@ def setup_indices(self):
         for name, ind in self.indices.items():
             setattr(self, name, ind)
 
+    def test_create_index_existing_name(self):
+
+        # this is testing for GH11193, when an existing index is passed, and a new name is not specified, the new index
```

Comment:
Thanks for the code review Jeff. I'm going to implement your suggestions.

Gianluca Rossi

Phone: +44 (0)7901 646 958
E-mail: gr.gianlucarossi@gmail.com
LinkedIn: http://uk.linkedin.com/in/gianlucarossi1986/en/

On 25 November 2015 at 14:39, Jeff Reback <notifications@github.com> wrote:

> In pandas/tests/test_index.py
> <https://github.com/pydata/pandas/pull/11695#discussion_r45873228>:
>
> > @@ -41,6 +41,18 @@ def setup_indices(self):
> >          for name, ind in self.indices.items():
> >              setattr(self, name, ind)
> >
> > +    def test_create_index_existing_name(self):
> > +
> > +        # this is testing for GH11193, when an existing index is passed, and a new name is not specified, the new index
>
> this needs to use self.create_index() which will be run for all index
> types
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pydata/pandas/pull/11695/files#r45873228>.
>